codespeed/commits/github.py

```diff
@@ -11,13 +11,16 @@
 try:
     # Python 3
     from urllib.request import urlopen
+    from urllib.request import Request
 except ImportError:
     # Python 2
-    from urllib import urlopen
+    from urllib2 import urlopen
+    from urllib2 import Request
```

Comment:
I didn't want to introduce additional dependency (``requests``), so I just followed the existing approach.